cpp/ql/src/semmle/code/cpp/Variable.qll

```diff
@@ -176,7 +189,21 @@ class Variable extends Declaration, @variable {
 }
 
 /**
- * A particular declaration or definition of a C/C++ variable.
+ * A particular declaration or definition of a C/C++ variable. For example in
+ * the following code there are five variable declaration entries - two for
+ * `a`, and one each for `b`, `c` and `d`:
+ * ```
+ * extern int a;
+ * int a;
+ *
+ * void myFunction(int b) {
+ *   int c;
```

Comment:
I think that `extern void myFunction(int b);` would make more sense, since there is a many-to-one relationship for functions.

Reply:
I don't understand the relevance of `FunctionDeclarationEntry`s.  There's only one `VariableDeclarationEntry` on that line.

Reply:
You're right, `FunctionDeclarationEntry`s are a separate issue.